Tools/PowershellModule/doc/WingetRestSource.md

```diff
@@ -1,575 +1,174 @@
+
+This section provides guidance on how to create a REST source for the Windows Package Manager. ISVs or Publishers may host and manage a REST source if they would like full control of the packages available in a source. An independently hosted source may choose to expose the read endpoints publicly, restrict access to specific IP address via the addition of a traffic shaping module or restrict access by Microsoft Entra Id authentication.  The basic setups configured by the cmdlets and examples result in a source that is publicly readable but requires an authorization key to manage. 
+
+Windows Package Manager offers a comprehensive package manager solution including a command line tool and a set of services for installing applications. For more general package submission information, see [submit packages to Windows Package Manager](https://learn.microsoft.com/windows/package-manager/package/).
+
+## Manage Windows Package Manager REST source with PowerShell
+
+To simplify the management and interaction with the Windows Package Manager REST source, the `Microsoft.WinGet.RestSource` PowerShell module has been made available. This PowerShell module provides cmdlets that enable you to Add, Find, Remove and Get package manifests from your Windows Package Manager REST source, as well as stand up a new Windows Package Manager REST source in Azure.
+
+The following steps are for managing a Windows Package Manager REST source with PowerShell:
+
+1. Download and install the `Microsoft.WinGet.RestSource` PowerShell Module.
+2. Automate the creation of a Windows Package Manager REST source.
+3. Publish Package Manifests to the Windows Package Manager REST source.
+4. Find Package Manifests from Windows Package Manager REST source.
+5. Retrieve published Package Manifests from the Windows Package Manager REST source.
+6. Remove published Package Manifests from the Windows Package Manager REST source.
+
+### PowerShell pre-Requisites
+
+Before getting started with the Windows Package Manager REST source with PowerShell, here are a few recommended steps you should complete before using the PowerShell module to ensure you can successfully stand up a new Windows Package Manager REST source in Azure.
+
+1. Open PowerShell as an Administrator
+2. Set the PowerShell execution policies
+    ```Powershell
+    PS C:\> Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+    ```
+4. Install the Azure Az module 
+    ```Powershell
+    PS C:\> Install-Module az -AllowClobber
+    ```
+5. Connect to Azure with an authenticated account
+    ```Powershell
+    PS C:\> Connect-AzAccount
+    ```
+6. Select the required subscription using Set-AzContext
+    ```Powershell
+    PS C:\> Set-AzContext -Subscription [Paste the Azure Subscription here]
+    ```
+
+### Download and install the PowerShell module
+
+#### Get PowerShell Module from PSGallery (Recommended)
+1. Install the PowerShell module from PSGallery

```

Comment:
Perhaps if the PowerShell module requires a minimum version of 7.4 to run, we can switch it to Install-PSResource, as it is built-in with this version :)

Reply:
I'm not sure how much pushback we'll get for not supporting Windows PowerShell, but it might not be worth the effort since setting up a REST source isn't likely to be a frequent need. I think I'm OK with PowerShell 7 as a requirement here.